### ISSUE TITLE
Linbasex error for even image sizes

### ISSUE THREAD

**Issue by DanHickstein** [PyAbel/PyAbel#202]
When I try running linbasex for an even image size, I get an unhelpful error:
```python
abel.Transform(np.random.random((100,100)), method='linbasex')
```
```
LinAlgError                               Traceback (most recent call last)
<ipython-input-12-a799ff17370b> in <module>()
----> 1 abel.Transform(np.random.random((100,100)), method='linbasex')
...
LinAlgError: Incompatible dimensions
```

Should we just check if the image dimensions are even, and, if so, raise a more helpful ValueError?

**Comment by stggh** [comment]
Good find.  Would you like me to add the odd width check for `linbasex.py` change in the current PR, or submit a new PR?

**Comment by DanHickstein** [comment]
I believe that this was addressed by #201, and can now be closed. (@stggh, please re-open if that's not the case.)
